js/pages/trackers.es6.js

```diff
@@ -67,6 +60,14 @@ Trackers.prototype = $.extend({},
                 template: ItemMenuTemplate
             });
 
+            this.views.trackerlist = new TrackerListView({
+                pageView: this,
+                model: new TrackerListModel({heading: 'Top Blocked', max:5}),
```

Comment:
If you need a switch because the template is shared and needs different headings, in the template you could have an if/else statement that prints a heading depending on some property of the view.